src/common/modal.js

```diff
@@ -26,7 +26,7 @@ export const alert = (title, body) => {
         <Header closeButton>
           <Title>{title}</Title>
         </Header>
-        <Body>{body}</Body>
+        {body && <Body>{body}</Body>}
```

Comment:
It is an alert, so sometimes, you just wanna say "Test passed !" and there are disgracious empty body separators under :)